components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -1521,6 +1575,7 @@ real(rtype) function polysvp1(t,i_type)
     ! REPLACE GOFF-GRATCH WITH FASTER FORMULATION FROM FLATAU ET AL. 1992, TABLE 4 (RIGHT-HAND COLUMN)
 
     !local variables
+    character(len=8), parameter :: subname = 'polysvp1'
```

Comment:
subname isn't used in polysvp1. Delete it?

Reply:
Originally I added `check_temp` call in both svp functions. Later I moved that call to `qv_sat` as both svp were called from `qv_sat`. I will move back those calls to svp functions and use this variable.

Reply:
I actually like the way you set things up and would rather not call check_temp from svp functions. I just want the error message in svp to be a bit more clear...